debian/enterprise-kafka/Dockerfile.rpm

```diff
@@ -30,7 +30,7 @@ EXPOSE 9092
 
 RUN echo "===> installing confluent-support-metrics ..." \
     && yum -q -y update \
-    && yum install -y \
+    && yum install -y --nogpgcheck \
```

Comment:
The `APT_ALLOW_UNAUTHENTICATED` var was intended for this but perhaps is too debian specific.